dart/gui/CMakeLists.txt

```diff
@@ -2,19 +2,20 @@
 file(GLOB srcs "*.cpp")
 file(GLOB hdrs "*.h")
 
+include_directories(SYSTEM ${OPENGL_INCLUDE_DIR})
 include_directories(SYSTEM ${GLUT_INCLUDE_DIR})
 
 # Add subdirectories
 if(HAVE_OPENSCENEGRAPH)
   add_subdirectory(osg)
 endif(HAVE_OPENSCENEGRAPH)
 
-set(dart_gui_hdrs ${dart_gui_hdrs} ${hdrs})
-set(dart_gui_srcs ${dart_gui_srcs} ${srcs})
+set(dart_gui_hdrs ${dart_gui_hdrs} ${hdrs} ${dart_renderer_hdrs})
+set(dart_gui_srcs ${dart_gui_srcs} ${srcs} ${dart_renderer_srcs})
 
 # Library
 dart_add_library(dart-gui ${srcs} ${hdrs})
```

Comment:
The errors come from the fact that we need to add the `dart_renderer_hdrs` and `dart_renderer_srcs` in the `dart_add_library` command too..